src/App.css

```diff
@@ -1,14 +1,27 @@
+/* Repaired text colour issue using a tag */
+
 .App {
   text-align: center;
-  color: rgb(180, 110, 4);
+  color: black;
+}
+/* changed the text colour -> blk header */
```

Comment:
Please remove comments, if you want to explain your changes, please drop a message on the PR

Reply:
Done. Thank you